testnet/launcher/fundsrecovery/docker.go

```diff
@@ -0,0 +1,105 @@
+package fundsrecovery
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/obscuronet/go-obscuro/go/common/docker"
+	"github.com/sanity-io/litter"
+)
+
+type FundsRecovery struct {
+	cfg         *Config
+	containerID string
+}
+
+func NewFundsRecovery(cfg *Config) (*FundsRecovery, error) {
+	return &FundsRecovery{
+		cfg: cfg,
+	}, nil // todo (@pedro) - add validation
+}
+
+func (n *FundsRecovery) Start() error {
+	fmt.Printf("Starting L2 contract deployer with config: \n%s\n\n", litter.Sdump(*n.cfg))
+
+	cmds := []string{
+		"npx", "hardhat", "obscuro:deploy",
```

Comment:
should this just be "deploy" actually? Looks like "obscuro:deploy" does some wallet extension shenanigans